google/resource_container_cluster_test.go

```diff
@@ -219,6 +231,10 @@ func testAccCheckContainerClusterDestroy(s *terraform.State) error {
 	return nil
 }
 
+var setFields map[string]struct{} = map[string]struct{}{
```

Comment:
Could this just be a list/set of strings rather than having an unused struct{} value?